README.md

```diff
@@ -28,6 +28,8 @@ gulp.task('psc', function(){
 });
 ```
 
+There is also [a fuller example](#full-example) that makes use of all the provided tasks in a common setup.
```

Comment:
This all looks great! Thanks for adding the example.

I don't mean to be nit-picky, but can we say "a more complete example" here instead of "a fuller example"?

Thanks!